src/card.css

```diff
@@ -0,0 +1,124 @@
+.card {
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+  width: 200px;
+}
+
+.card__photo {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 0;
+}
+
+.card__name {
+  width: auto;
```

Comment:
auto is default width's property, so you don't need to use it like that, it has no effect. If you need to use width somewhere you can try with % values, however I think deleting this won't have any effect